examples/systemd/fdo-owner-onboarding-server.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=FDO owner onboarding server
+After=network-online.target
+
+[Service]
+Environment=LOG_LEVEL=trace
```

Comment:
I don't think we should have `LOG_LEVEL=trace` by default. I think this should be `info`. (We have an outstanding issue to check all log levels).

Reply:
I'll move these to `info`